connect/api/src/main/java/org/apache/kafka/connect/health/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Provides interface for describing the state of a running Connect cluster.
```

Comment:
We should be clear about who is going to be describing the cluster, and how this information can be used:

```suggestion
 * Provides an API for describing the state of a running Connect cluster.
 * The Connect runtime will instantiate members of this package and expose them to
 * components whose API gives them access to cluster information.
 * <p/>
 * For example, {@link org.apache.kafka.connect.rest.ConnectRestExtension REST extensions}
 * can use their {@link org.apache.kafka.connect.rest.ConnectRestExtensionContext#clusterState() context}
 * to learn about the state of the cluster.
```